src/test/resources/assets/alwaysmoreitemstest/stationapi/lang/en_US.lang

```diff
@@ -22,3 +22,5 @@ alwaysmoreitems.actionButton.@.cheat_dummy=Cheaty Dummy
 description.@.wooden.door.1=Wooden Doors allow you to block monsters from entering your building.\nTesting sentences.
 description.@.wooden.door.2=Clicking on a door changes its state from open to closed and vice versa.
 description.@.wooden.door.3=Wooden Doors can be opened/closed via Redstone Circuits.
+
+multiblock.alwaysmoreitems.test=Test Multiblock
```

Comment:
This is inconsistent. Should be `multiblock.modid.multiblockname` or `multiblock.alwaysmoreitems.modid.multiblockname`.